src/application/hook.ts

```diff
@@ -1,21 +1,30 @@
 import { useRecoilState } from 'recoil';
-import { dataState, loadState } from './state';
-import { Item } from './state';
+import { itemsState } from './state';
+import { useEffect } from 'react';
+import listItems from '../infrastructure/api/fetch_Items';
 
-const useItemFetch = () => {
-  const [data, setData] = useRecoilState<Item[]>(dataState);
-  const [showMore, setShowMore] = useRecoilState<number>(loadState);
+const useItems = () => {
+  const [data, setData] = useRecoilState<any>(itemsState);
+  let tempItemHolder;
```

Comment:
always avoid using dynamic variables liek this that can change. 

Instead, use array destructuring to add the new items to the old ones